editor/components/inserter/menu.js

```diff
@@ -176,6 +184,18 @@ export class InserterMenu extends Component {
 				/>
 
 				<div className="editor-inserter__results">
+					{ !! childItems.length &&
```

Comment:
Can we extract this code to a new component to avoid confusing conditional checks for an icon and title? As far as I understand when there are child blocks then the icon and title must be set, right?

Reply:
Yes if we have childblocks the icon and title are set unless the block did not use a title/item, but that's part of the block responsibility.

Reply:
The child blocks rendering logic was extracted to a separate component.